tests/spyre_util.py

```diff
@@ -40,6 +41,10 @@ def patch_environment(use_cb: bool,
     monkeypatch.setenv("VLLM_SPYRE_DYNAMO_BACKEND", backend)
     monkeypatch.setenv("VLLM_SPYRE_USE_CHUNKED_PREFILL",
                        "1" if use_chunked_prefill else "0")
+    # NB: setting this env var explicitly is needed to set the desired value for
+    # the chunk size in the case that granite 8b TP4 is detected
```

Comment:
In that case wouldn't we also need to re-override the internal config for `max_num_batched_tokens` as well?

Reply:
very hot 🌶️